src/Omega_h_mesh.cpp

```diff
@@ -1105,7 +1105,7 @@ LOs nodes_on_closure(
   return collect_marked(nodes_are_on);
 }
 
-#ifdef OMEGA_H_USE_CUDA
+#ifdef KOKKOS_ENABLE_CUDA
```

Comment:
I'd like to only use `KOKKOS_ENABLE_CUDA` where absolutely necessary.  

With 'recent' cuda versions, is adding `__host__` still needed?  I'm guessing there will be a compiler warning if it is needed.